appcues/src/main/java/com/appcues/data/mapper/step/primitives/SpacerPrimitiveMapper.kt

```diff
@@ -1,10 +1,9 @@
 package com.appcues.data.mapper.step.primitives
 
-import com.appcues.data.mapper.styling.mapComponentStyle
 import com.appcues.data.model.ExperiencePrimitive.SpacerPrimitive
 import com.appcues.data.remote.response.step.primitive.PrimitiveResponse.SpacerPrimitiveResponse
 
 internal fun SpacerPrimitiveResponse.mapSpacerPrimitive() = SpacerPrimitive(
     id = id,
-    style = style.mapComponentStyle(),
+    spacing = spacing,
```

Comment:
I see this matches iOS now https://github.com/appcues/appcues-ios-sdk/blob/main/Sources/AppcuesKit/Data/Models/Experience/ExperienceComponent.swift#L238

Looks like we should update the spec to include the `spacing` property? https://github.com/appcues/appcues-mobile-experience-spec/blob/main/experience-schema.json#L480-L490

Reply:
yes